src/opm/output/eclipse/Summary.cpp

```diff
@@ -4031,7 +4054,9 @@ namespace {
 
         default:
             throw std::logic_error {
-                fmt::format("make_default_nodes does not yet support: {}", keyword)
+                fmt::format(std::string("ACTIONX/UDQ using summary keywords (here {}) of category ")+
```

Comment:
Thanks, I knew there was a way, but could not recall it.

What is not supported is the whole category (i.e. all keywords). I just list the keyword for additional information. That does not seem reflected in your version? Maybe
```
fmt::format("Category '{}' (e.g. of summary keyword {} used here) is not supported in ACTIONX/UDQ",
            category, keyword);
```

On second thought I would skip UDQ because it is not involved.

Reply:
> I just list the keyword for additional information. That does not seem reflected in your version?

Ah, right.  In that case I'd probably phrase it as
```C++
fmt::format("Keyword category '{}' (e.g., summary keyword {}) is not supported in ACTIONX",
            category, keyword);
```
instead.